src/crypto/tls/handshake_client_tls13.go

```diff
@@ -450,6 +494,7 @@ func (hs *clientHandshakeStateTLS13) readServerParameters() error {
 		c.sendAlert(alertUnexpectedMessage)
 		return unexpectedMessageError(encryptedExtensions, msg)
 	}
+
```

Comment:
Remove this spurious new line.

Reply:
This isn't spurious -- I wanted to separate them with whitespace.

Reply:
Our policy has been to avoid these kinds of changes, since they make rebasing harder.